docs/content/guides/operations/providers/aws-provider/howto-aws-provider-irsa/index.md

```diff
@@ -0,0 +1,130 @@
+---
+type: docs
+title: "How-To: Configure the AWS cloud provider with IAM Roles for Service Accounts (IRSA)"
+linkTitle: "AWS provider with IAM Roles for Service Accounts(IRSA)"
+description: "Learn how to configure the AWS provider with IAM Roles for Service Accounts(IRSA) for your Radius Environment"
+weight: 300
```

Comment:
```suggestion
weight: 200
```

Reply:
weight determines the order of the page under a section. can you also change the weight to 100 for the AWS provider access key page

Reply:
done